client/package.json

```diff
@@ -88,18 +88,33 @@
         },
         "bitbake.pathToEnvScript": {
           "type": "string",
-          "default": "${workspaceFolder}/sources/poky/oe-init-build-env",
+          "examples": ["${workspaceFolder}/sources/poky/oe-init-build-env"],
```

Comment:
I don't see any of the examples you set when navigating in vscode settings. Is it normal?

Reply:
![Screenshot from 2023-11-20 10-55-27](https://github.com/savoirfairelinux/vscode-bitbake/assets/92585455/13140d1e-978c-437e-969c-c6f6492a92ed)

Reply:
![image](https://github.com/savoirfairelinux/vscode-bitbake/assets/19970889/b8e86a0c-5cf3-45fa-8054-88a8f9f432fb)
They properly render if I ask VSCode for completions, using Ctrl+Space

Reply:
Thanks for your screenshot, it seems examples are not reported in the settings view, with no way to see them. Guess I'll have to add them to the description. I could open an issue on vscode for that...

Reply:
I pushed two commits to address this issue and clarify the doc.